### DISCUSSION TITLE
switches acting up after install

### DISCUSSION THREAD

**Discussion by SaWkYmOd** [BottlecapDave/HomeAssistant-OctopusEnergy#218] [General]
Installed this and enabled it and all ok for a while then my dashboard doesnt update anymore, For instance a switch will turn on but 1 sec later it automatically goes to the off position but the switch is actually still on...If i disable the the interigation and restart, everything goes back to normal..... Any ideas??

**Comment by BottlecapDave** [comment]
No switches are exposed via this integration. I assume you mean binary sensors? If so, this is odd behaviour and not what I'd expect. I've been seeing the Octopus Energy API being quite unstable recently and wonder if this is having an effect. Do you have any errors in your logs?

**Reply by SaWkYmOd** [reply]
no im on about my other switches on my dashboard, the sensors I use from you are fine , it just seems to mess other things up like everything else on my dash ie light switches etc, i have to restart HA and it goes back to normal for a while then messes up again....there seems to be no logs for it either.  Ive had it disabled all day and had no problems since...Strange.... Ill clear my logs re enable it and see if anything pops up......Thanks for the quick reply

**Reply by BottlecapDave** [reply]
The integration shouldn't be having any effect on other integrations as it follows the best practices laid out by HA. I also haven't seen anyone else report such issue. Things that might cause this behaviour on your device include
* memory constraints on your device
* cpu constaints on your device
* too many integrations for your device

For instance I used to run my install on a Raspberry Pi 3 (2GB) and I had memory pressures after I installed too many integrations which caused my instance's frontend to become unresponsive. I had to check [memory stats](https://www.home-assistant.io/integrations/systemmonitor/) and restart my instance until I upgraded to a 4GB memory.

**Comment by SaWkYmOd** [comment]
After some digging its looking like its the tuya interigations thats playing up , lots of people reporting same issue
Thanks for your help and great work btw.... hope they sort it soon

**Reply by BottlecapDave** [reply]
I'm glad it wasn't a problem with this integration and hope they find a solution soon :)
